### ISSUE TITLE
Any plan to integrate with polkadot.js apps?

### ISSUE THREAD

**Issue by xlc** [paritytech/polkadot_network_directory#23]
polkadot.js apps currently have a quite complete list of networks https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints

It will be great if the effort can be combined

**Comment by chevdor** [comment]
Thanks for the link @xlc. This is still a little early as we are still tweaking schemas but that would be good indeed.

I think however that the strategy will differ. The data you point out is rather centralized in the [repo]([https://github.com/polkadot-js](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints)). That means that endpoints added/removed by Jaco will affect all the users down the line without them having much choice.

I would like to see each "rpc provider" able to provide their own list (on IPFS even maybe...) and let users aggregate this data into their own curated local list. That would also allow projects like yours to have their own list of internal rpc providers.

I am working on a tool that will take care of this aggregation and more. More on that later and as soon as I can publish something useful, I really would like indeed to either make use of the list you mentioned, or as another option, help build this list.

As a side note, we opened the repo to get things started and added lots of info in the YAML that is data we could fetch online using either an rpc endpoint or an api service. The goal is to go away from stuffing those yaml files manually and fetch as much data as possible, based on the endpoints provided in the yaml.. or other solutions such as what I mentioned above. Stay tuned and thanks for feeding ideas :)

**Comment by chevdor** [comment]
And here is a very early version of the tool I mentioned above: https://github.com/chevdor/subrpc

**Comment by Polkadot-Forum** [comment]
This issue has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/subrpc-manage-multiple-endpoint-registries-and-quickly-find-endpoints/1748/4

**Comment by chevdor** [comment]
When this issue was initially opened, `subrpc` was not published yet, now that [it is](https://github.com/chevdor/subrpc), I think the right approach would be to use **some** registries (including the one provided by this repo) to generate the data used by PolkadotJS. 

I think users go used to adding their endpoint to the PolkadotJS repo so PolkadotJS may also created its own registry. That's an easy PR anyone could create.

I will close this issue as there is nothing to be done in this repo. I would invite creating an issue/PR in the PolkadotJS repo although as I mentioned in the thread mentioned above, I think we don't have yet the data quality we would want. 

In order to get there, we need more paras in this repo and more maintained registries in general. Having that and `subrpc`, it will be rather trivial to use the aggregated data in PolkadotJS.

tldr:
- `subrpc` is there to help aggregate data
- a dataset can be created by any "user" such as PolkadotJS or anyone really
- a dataset should be created by any rpc endpoint provider
